### PR TITLE
Updated mining vendor items - diet edition

### PR DESCRIPTION
**Pull request by PerishedFraud**
Updates mining vendor items. Some prices are more reasonable now. The order now makes sense. Adds some new items which will be covered below.

Additionally you can now eat through the explorer mask without cutting off internals, and the mining conscription kit comes with a mining ID.

**Why?** - The old item selection was outdated and looked bad, and items with little, inferior or niche uses were overpriced too.

**Diet edition?** - The earlier version included pricy xenobio potions. People went ape over it. So did I. The more OP aspects of miner will be adressed before similar stuff is attempted again. Can't have both it seems. Look forward to my tendril unfucking PR.

[Image of new item selection.](https://i.imgur.com/HSqf0Qj.png) (edit: resonator is 750 and super resonator 1500 now)

**Individual item changes discussed below (minor price changes not covered):**
Marker beacons - Lowered cost and removed redundant options. These saw almost no use. Now they're cheaper on you and in bulk.
Added burn and advanced healing kit - Lavaland has plenty of burn damage to go around, yet only brute kit was available.
Greatly lowered prices of resonators and pickaxes to reflect their inferiority and lack of use.
Added expanded oxygen tank. This thing sees use with bone armor, and would normally be aquired from legion corpses. This is a non-RNG alternative.
Added a survival knife. No major reason beyond these being strangely unobtainable (though bone dagger is the same item really).
Added pizza as a food option

**Other changes:** The explorer mask now allows you to eat without cutting off internals. This feature was previously exclusive to the welding mask iirc. It makes sense for the miner mask to have this considering the guys are on internals 24/7. The conscription kit now comes with an ID becuase some of the ghost spawns you could buy the kit for had no ID to start with, thus rendering the ID upgrade in the kit worthless. Assuming you'd want to bring them up, they'd be forced to pester the HoP or snag an ID from somewhere, which is bleh.

**Clarity changes include:** KC and KA now include proto in their names in the vendor. Streamlined KA upgrade names. Space cash now says that it's 1000. Stabilizing serum renamed to regen. core stabilizer to immediately show its purpose. Nanotrasen minebot renamed to mining bot companion. Why did it ever have NT in its name? Surely you won't buy syndie minebots from this. Chassis reskin modules now clearly say they're cosmetic. Alien toy>Toy alien. (Note, these are only what the items appear as in the vendor. Actual names unchanged)

## Changelog
:cl:
add: Some new items in the mining vendor.
tweak: Some prices changed in the mining vendor, can also eat through explorer mask.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Another PR is going to get merged that buffs the resonator which should be taken into account when reducing the prices
https://github.com/BeeStation/BeeStation-Hornet/pull/2781

**Comment by PerishedFraud** [comment]
Updated accordingly. That plasma cutter nerf is overkill btw. Go figure.

**Comment by VictorVow** [comment]
Expanded emergency oxygen tank should also be available in the techweb if miners can buy it imo.

**Comment by PerishedFraud** [comment]
rly should, though that has little to do with this PR. advanced engineering should give it, probably.

File: code/modules/mining/machine_vending.dm [hunk 1]
```diff
@@ -12,56 +12,58 @@
...
-		new /datum/data/mining_equipment("Mining Conscription Kit",		/obj/item/storage/backpack/duffelbag/mining_conscript,				1000),
-		new /datum/data/mining_equipment("Jetpack Upgrade",				/obj/item/tank/jetpack/suit,										2000),
-		new /datum/data/mining_equipment("Space Cash",					/obj/item/stack/spacecash/c1000,									2000),
+		new /datum/data/mining_equipment("Advanced Ore Scanner",		/obj/item/t_scanner/adv_mining_scanner,								800),
+		new /datum/data/mining_equipment("Explorer's Webbing",			/obj/item/storage/belt/mining,										500),
+		new /datum/data/mining_equipment("Jump Boots",					/obj/item/clothing/shoes/bhop,										2000),
+		new /datum/data/mining_equipment("Proto-Kinetic Crusher",		/obj/item/twohanded/kinetic_crusher,								800),
+		new /datum/data/mining_equipment("Proto-Kinetic Accelerator",	/obj/item/gun/energy/kinetic_accelerator,							500),
+		new /datum/data/mining_equipment("Resonator",					/obj/item/resonator,												7500),
+		new /datum/data/mining_equipment("Upgraded Resonator",			/obj/item/resonator/upgraded,										1500),
```
Comment (PowerfulBacon) [inline code/modules/mining/machine_vending.dm]: 7500 for a normal resonator, 1500 for an upgraded one

Comment (PerishedFraud) [reply code/modules/mining/machine_vending.dm]: The delete key has been thoroughly beaten and likely won't betray us again.

**Comment by MitztheKat** [comment]
YES
YES
Y E S

**Comment by PerishedFraud** [comment]
well?

**Comment by ike709** [comment]
image

**Pull request merged**